### PR TITLE
Feat/17 what we do section

### PR DESCRIPTION
**Pull request by yunshen513**
### Descriptions

- Link to UI design (if possible):

### Provide screenshots or videos that the function is working.
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Comment by Cameron933** [comment]
Nice content!
Just one small thing:
image

**Review by Cameron933 — APPROVED** [review]

File: __tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { render, screen } from "@testing-library/react";
```
Comment (tatoMa) [inline __tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx:1]: WhatWeDoItem.test.tsx inside of the WhatWeDoItem folder is not a proper way of this file location.

File: components/general/Hero.tsx [hunk 2]
```diff
@@ -13,7 +13,10 @@ const Hero = () => {
             digital agency
             <br /> from Melbourne.
           </div>
-          <div className={`${textFont.className} global__text-xlarge ${styles["hero__txt"]}`}>
+          <div
+            className={`${textFont.className} global__text-xlarge ${styles["hero__txt"]}`}
+            aria-label="hero__txt"
```
Comment (tatoMa) [inline components/general/Hero.tsx:18]: You use this aria-label in the wrong way. 
1. I don't see any reason to use aria-label on this div.
2. aria-label should be a description or the meaning of this element, computer or mobile phone can not read text "hero__txt".

Comment (tatoMa) [inline components/general/Hero.tsx:18]: Why did you change this hero component? Is it related to your card?

File: __tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx (hunk 1 above)
Comment (yunshen513) [reply __tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx:1]: I just follow the way that previous commits did, like Button.test.tsx in Button folder. Should I just put test files under __test__/components ?

File: components/general/Hero.tsx (hunk 2 above)
Comment (yunshen513) [reply components/general/Hero.tsx:18]: Yes, it's not a very proper way. Any suggestions to select this element except getByText since this text is repeating all over the place. Is getAllByText[index] and getByTitle better? I'm not sure about this.

Comment (yunshen513) [reply components/general/Hero.tsx:18]: It caused testing failed when others add a new section with the same text, so I changed it. I will remove my changes.

File: __tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx (hunk 1 above)
Comment (tatoMa) [reply __tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx:1]: > 
Blindly copying and pasting is not always good and right. When WhatWeDoItem.test.tsx inside of the same name folder, it duplicated and caused confusion for other devs. Consider using "WhatWeDoItem/index.tsx" or just without same name parent folder.

File: components/general/Hero.tsx (hunk 2 above)
Comment (tatoMa) [reply components/general/Hero.tsx:18]: You may have missed the Testing playground tutorial.

You can find the recorded video on my Youtube channel or try this tool to see how it goes.
https://testing-playground.com/

Comment (tatoMa) [reply components/general/Hero.tsx:18]: If it is a problem unrelated to your card, consider creating a new bug ticket. Otherwise, change it if it is truly necessary (not recommended).

**Review by tatoMa — APPROVED** [review]
LGTM
